Lesson_09/homeworks_function.py

```diff
@@ -0,0 +1,15 @@
+
+# lesson_06, homework_06 task 6.1
```

Comment:
Your PR should no contain code, that is commented and does not used for any purposes - please delete unnecessary code.